src/blocks/icon/test/icon.cypress.js

```diff
@@ -166,6 +166,7 @@ describe( 'Test CoBlocks Icon Block', function() {
 		helpers.editPage();
 	} );
 
+	// pointer-events: none does not allow us to click
```

Comment:
Feel free to remove the comment if you are not able to reproduce the failing test.